workflow-fingerprint-compare/README.md

```diff
@@ -0,0 +1,72 @@
+# fingerprintCompare
+
+Workflow to generate jaccard matrices for all projects.
+
+## Overview
+
+## Dependencies
+
+* [perl 5.30](https://www.perl.org/)
+
+
+## Usage
+
+### Cromwell
+```
+java -jar cromwell.jar run fingerprintCompare.wdl --inputs inputs.json
+```
+
+### Inputs
+
+#### Required workflow parameters:
+Parameter|Value|Description
+---|---|---
+`finFile`|File|Input .fin file containing list of .fin files
+
+
+#### Optional workflow parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+
+
+#### Optional task parameters:
+Parameter|Value|Default|Description
+---|---|---|---
+
+
+### Outputs
+
+Output | Type | Description
+---|---|---
+`outputFin`|File|generated jaccard matrix
```

Comment:
output is a matrix, not a fin file
I suggest just calling this parameter : output